### ISSUE TITLE
In documentation for MRL Keyboard Navigation the tab navigation should be removed

### ISSUE THREAD

**Issue by nrobakova** [IgniteUI/igniteui-docfx#1958]
Open [dock](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/grid/multi_row_layout.html#keyboard-navigation)
 
There are tab and Shift+Tab combinations which are not supported in version 9.1:
![image](https://user-images.githubusercontent.com/11463104/82651362-bbb24680-9c24-11ea-859c-8e1962ef0c09.png)

**Comment by ChronosSF** [comment]
Duplicate of #1935
